controllers/useraccount/useraccount_controller.go

```diff
@@ -615,6 +614,14 @@ func (r *Reconciler) lookupAndDeleteCheUser(logger logr.Logger, config membercfg
 		return nil
 	}
 
+	if config.Che().IsDevSpacesMode() {
+		err := r.deleteDevSpacesUser(logger, userAcc)
+		if err != nil {
+			logger.Error(err, "DevSpaces user deletion failed", "user account", userAcc.Name)
+		}
+		return nil // log error but do not fail deletion if the DevSpaces db cleanup failed, treat it as a best effort
```

Comment:
if deletion failed and the record is still in the database this would mean that after reprovisioning user will not be able to use the database. Can we stop the user deprovioning in this case and log the error instead?

Reply:
Done in 71bf1730fb24cb6d4ad626cf50ccb1ed76ab7ea6